openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AdvanceSearch.spec.js

```diff
@@ -175,9 +175,6 @@ describe('Advance search should work properly for Add Group functionality', () =
     it(`Verify Add group functionality for All with ${operator.name} operator & condition ${CONDITIONS_MUST.contains.name} and ${CONDITIONS_MUST_NOT.notContains.name} `, () => {
       Object.values(FIELDS).forEach((field) => {
         let val = field.searchCriteriaSecondGroup;
-        if (field.owner) {
-          val = field.searchCriteriaSecondGroup.split(' ')[0];
-        }
```

Comment:
why we are removing this?

Reply:
earlier we were checking if only name of the user is present in the API call. Example for 'Aaron Johnson' we were asserting only 'Aaron'. I removed that ant introduced encodeURL to check the whole thing.